pkg/virt-launcher/virtwrap/api/converter.go

```diff
@@ -926,6 +926,12 @@ func Convert_v1_VirtualMachine_To_api_Domain(vmi *v1.VirtualMachineInstance, dom
 			Index: "0",
 			Model: "none",
 		})
+	} else {
+		domain.Spec.Devices.Controllers = append(domain.Spec.Devices.Controllers, Controller{
+			Type:  "usb",
+			Index: "0",
+			Model: "qemu-xhci",
```

Comment:
should we only be defining the usb controller if a usb device exists?

Reply:
yes, that is what we do. Note the `Model: "none"` above.